src/nupic/types/Types.hpp

```diff
@@ -44,7 +44,11 @@ namespace nupic {
 /**
  * Represents a 8-bit byte.
  */
-typedef int8_t Byte;
```

Comment:
what is the advantage to original `char` here? What is this Byte used to? (uint_8_least/fast_t ...?)

Reply:
Pybind11 automatically converts based on data type.  `char` array converts to/from unicode strings, `int8` converts to/from integers.